### PR TITLE
Maint

### PR DESCRIPTION
**Pull request by fgrunewald**
Collection of maintenance things. This PR fixes:

- #30 #16 bond orders are now cleared after an atom has been parsed; this also fixes issue #16 
- #15 is fixed and now also takes into account bond orders; in addition, new tests cover previously untested error
- [#394](https://github.com/marrink-lab/polyply_1.0/issues/394) from polyply is fixed; the atom names are now updated both in the meta-graph and molecule if a meta graph is provided. This fix also removes the need for calling the naming function twice
-

### CODE REVIEW

File: cgsmiles/tests/test_cgsmile_parsing.py [hunk 1]
```diff
@@ -269,6 +269,30 @@ def test_read_cgsmiles(smile, nodes, charges, edges, orders):
...
+                         "[CH]=[CH]",
+                        {0: ["$a1"], 1: ["$c1"]},
+                        None,
+                        None),
+                        # multiple non-one bonding l
+                        ("CC=[$a]=[$b]CC",
+                         "CCCC",
+                        {1: ["$a2", "$b2"]},
+                        None,
+                        None),
```
Comment (pckroon) [inline cgsmiles/tests/test_cgsmile_parsing.py:295]: Maybe add `"CC[$a]=[$b]CC"`?

**Review by pckroon — CHANGES_REQUESTED** [review]
Looks good, small questions/comments

File: cgsmiles/pysmiles_utils.py [hunk 2]
```diff
@@ -67,18 +67,32 @@ def rebuild_h_atoms(mol_graph, keep_bonding=False):
...
-               "is not considered aromatic. For example, 4-methyl imidazole "
-               "is often written as [nH]1cc(nc1)C, but should be written as "
-               "[NH]1C=C(N=C1)C. A corresponding CGSmiles string would be "
-               "{[#A]1[#B][#C]1}.{#A=[>][<]N,#B=[$]N=C[>],#C=[$]C(C)=C[<]}")
+        msg = (r"Likely you are writing an aromatic molecule that does not "
+               r"show delocalization-induced molecular equivalency and thus "
+               r"is not considered aromatic. For example, 4-methyl imidazole "
+               r"is often written as [nH]1cc(nc1)C, but should be written as "
+               r"[NH]1C=C(N=C1)C. A corresponding CGSmiles string would be "
+               r"{[#A]1[#B][#C]1}.{#A=[>][<]N,#B=[$]N=C[>],#C=[$]C(C)=C[<]}")
```
[the remark this answers is not in the document]
Comment (fgrunewald) [reply cgsmiles/pysmiles_utils.py:75]: ups leftover from something I tried

**Review by pckroon — APPROVED** [review]
LGTM!

**Pull request merged**